api/app/controllers/spree/api/v2/storefront/stores_controller.rb

```diff
@@ -7,8 +7,27 @@ def current
             render_serialized_payload { serialize_resource(current_store) }
           end
 
+          def export_google_rss_feed
+            settings
+            if @settings.nil?
+              raise ActionController::RoutingError, 'Not Found'
+            elsif @settings.enabled
+              send_data export_google_rss_service.value[:file], filename: 'products.rss', type: 'text/xml'
+            else
+              send_data 'Export not enabled for those settings.'
```

Comment:
If it's not enabled, I'd just send 404 to the outside world, just as if it didn't exist.